Makefile

```diff
@@ -54,21 +54,27 @@ lint: deps ## Concurrently runs a whole bunch of static analysis tools
 	$(call header,"Running a whole bunch of static analysis tools")
 	golangci-lint run
 
+GROUP_VERSIONS:="istio:v1alpha1"
 .PHONY: codegen
 codegen: install-operator-sdk ## Generates operator-sdk code
 	$(call header,"Generates operator-sdk code")
 	$(CUR_DIR)/bin/operator-sdk generate k8s
+	$(call header,"Generates clientset code")
+	GOPATH=$(shell echo ${GOPATH} | rev | cut -d':' -f 1 | rev) ./vendor/k8s.io/code-generator/generate-groups.sh client \
+		$(PACKAGE_NAME)/pkg/client \
+		$(PACKAGE_NAME)/pkg/apis \
+		$(GROUP_VERSIONS)
```

Comment:
@aslakknutsen Now we have both `operator-sdk` CLI which provides "deepcopy" generator and this which can also provide the same. Do we want to keep that as is or unify to only use `./vendor/k8s.io/code-generator/generate-groups.sh`

Reply:
hmm... Any advantage of using the Generated client vs just a client.Client? 'Type safe' Get?

Reply:
"type safe" get/create/delete :)

plus dogfooding potentially used stuff by other "clients"